server/controllers/stock/reports/stock_entry.report.handlebars

```diff
@@ -125,21 +149,30 @@
 
           <!-- no data -->
           {{#if entryFromDonation.isEmpty}}
-            <tr><th class="text-center">{{translate 'STOCK.NO_DATA'}}</th></tr>
+            <tr><th colspan="6" class="text-center">{{translate 'STOCK.NO_DATA'}}</th></tr>
           {{/if}}
         </table>
       {{/if}}
 
+      <br>
+
       <!-- stock entry to other depot -->
       {{#if includeTransferEntry}}
-        <h3>{{translate 'STOCK.RECEIPT.ENTRY_DEPOT'}}</h3>
-
         <table class="table table-condensed table-report">
+          <thead>
+            <tr style="background-color: #EFEFEF;">
+              <th colspan="3"><h3 style="margin: 0px;">{{translate 'STOCK.RECEIPT.ENTRY_DEPOT'}}</h3></th>
```

Comment:
It looks like this key might not be defined correctly.

![image](https://user-images.githubusercontent.com/896472/83439410-fd1fce80-a43a-11ea-89bf-861890463589.png)


Shouldn't this say something about from other depot?